hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -230,29 +335,19 @@ def publish_files(self, task_settings, item, publish_path):
         if start_frame and end_frame:
             alembic_args.append("-fr %d %d" % (start_frame, end_frame))
 
-        # Creating a temporary file on the publish path, where the alembic from maya would be
-        # exported
-        publish_file_temp = tempfile.NamedTemporaryFile(mode='w+b',
-                                                        suffix='.abc',
-                                                        prefix='tmp',
-                                                        dir="/var/tmp/",
-                                                        delete=True)
-
-        publish_path_temp = publish_file_temp.name.replace("\\", "/")
-
         # Set the output path:
         # Note: The AbcExport command expects forward slashes!
-        alembic_args.append("-file %s" % publish_path_temp)
+        alembic_args.append("-file %s" % publish_path)
 
         # Set the root node to be exported
         alembic_args.append("-root %s" % item.get_property("lod_full_name"))
 
         # Add args based on publish settings
-        if task_settings["Export UVs"].value:
+        if task_settings.get("Export UVs").value:
```

Comment:
There is no value in using `get` over `[]` to access the setting.  If `get` returns None, we would get an error saying NoneType object has no attribute value.

Reply:
Addressed.